### PR TITLE
Issue 83 - CharField and TextField should generally not be nullable

### LINKED ISSUE #83: CharField and TextField should generally not be nullable
Related to 9cf101954. I fixed `templates/event.html` with a `default_if_none:''` filter to avoid invalid CSS:

``` html
<style>
    None
</style>
```

but this really should have been enforced in the model, since it doesn’t really make sense to distinguish between `None` and `''` (empty string) for this attribute. The same can be said with `EventPage`’s `title`, `description`, and many other fields holding text contents.

### PR DESCRIPTION
**Pull request by laidibug**
Fixes #83 

Just general cleanup on the fields in regards to the nulls in the text based fields.
The migrations have been added and tests adjusted to deal with empty strings.

### CODE REVIEW

File: jobs/models.py [hunk 1]
```diff
@@ -257,9 +253,7 @@ class Meetup(PublishFlowModel):
     city = models.CharField(max_length=255)
```
Comment (MarysiaLowas) [inline jobs/models.py]: Please delete `null=True` for the _website_ attribute as well.

File: jobs/models.py [hunk 2]
```diff
@@ -206,8 +204,7 @@ class Job(PublishFlowModel):
     cities = models.CharField(max_length=255)
```
Comment (MarysiaLowas) [inline jobs/models.py]: Please delete `null=True` for the _website_ attribute as well.

File: applications/migrations/0021_auto_20161031_0517.py [hunk 3]
```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
```
Comment (MarysiaLowas) [inline applications/migrations/0021_auto_20161031_0517.py]: Could you merge both migrations for _applications_ into one?

File: core/migrations/0022_auto_20161031_0433.py [hunk 4]
```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
```
Comment (MarysiaLowas) [inline core/migrations/0022_auto_20161031_0433.py]: Could you merge both migrations for _core_ into one?

**Comment by olasitarska** [comment]
Closing due to inactivity. Please re-open if you still decide to continue to work on this!

**Pull request closed without merging**